tracer/src/Datadog.Trace/Debugger/Sink/NonBatchUploader.cs

```diff
@@ -0,0 +1,18 @@
+// <copyright file="NonBatchUploader.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Datadog.Trace.Debugger.Sink;
+
+internal class NonBatchUploader : IBatchUploader
```

Comment:
nit: I would call this `NullBatchUploader`, as that matches other "`Null`" implementations we have. To me, `NonBatchUploader` implies that it _does_ still do the upload, it's just "non-batching" 😄

Reply:
I agree with you on this. @shurivich can you do it in a separate PR?